app-backend/db/src/test/scala/com/azavea/rf/database/AnnotationProjectDaoSpec.scala

```diff
@@ -332,4 +335,116 @@ class AnnotationProjectDaoSpec
       )
     }
   }
+
+  test("list projects belonging to a campaign") {
+    check {
+      forAll(
+        (
+            userCreate: User.Create,
+            annotationProjectCreates: List[AnnotationProject.Create],
+            taskFeaturesCreate: Task.TaskFeatureCollectionCreate,
+            campaignCreate: Campaign.Create
+        ) => {
+          val pageSize = 20
+          val pageRequest = PageRequest(0, pageSize, Map.empty)
+
+          val listIO = for {
```

Comment:
This test would be better if either there were two lists of annotation project creates or if only some of the projects in the list had the campaign attached. As written, if the campaign filtering didn't work but just listed all the annotation projects, it would still pass.